replica/database.hh

```diff
@@ -533,6 +533,10 @@ private:
     bool _is_bootstrap_or_replace = false;
     sstables::shared_sstable make_sstable(sstables::sstable_state state);
 
+    // The resize phase can be found in two places, in group0 (the authoritative source) and in local table state.
+    // Once table completes splitting work, the phase in table state will mirror the one in group0, so the
+    // coordinator knows this table is ready for current (not a stale; previously revoked) split request.
+    api::timestamp_type _resize_phase = {};
```

Comment:
`{}` is a bad default, there is min_timestamp.

Reply:
but why is it called a phase?

Reply:
Borrowed this term from the doc. It's better to call it resize_timestamp, right? as it denotes the time stamp of the resize, not a phase (stage). @tgrabiec what do you think?

Reply:
was it changed?